docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,123 @@
+---
+comments: true
+---
+
+# Add a metrics provider endpoint
+
+In this guide, we will create a placeholder provider.
+The following steps are a starting point to create your own custom provider:
+<!-- markdownlint-disable MD007 -->
+
+1. Fork and clone the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
+  for more information [checkout this link](https://keptn.sh/stable/docs/contribute/general/git/fork-clone/)
```

Comment:
```suggestion
1. **Fork and clone** the
[Keptn repository](https://github.com/keptn/lifecycle-toolkit).
 For more information, see
 [Fork and clone the repository](https://keptn.sh/stable/docs/contribute/general/git/fork-clone/).
```
This needs to be two sentences.  And you bold the short name of the other steps so let's be consistent here.